make.js

```diff
@@ -560,6 +564,16 @@ target.bump = function() {
         }
 
         taskJson.version.Patch = taskJson.version.Patch + 1;
+        taskLocJson.version.Patch = taskLocJson.version.Patch + 1;
+
         fs.writeFileSync(taskJsonPath, JSON.stringify(taskJson, null, 4));
+        fs.writeFileSync(taskJsonPath, JSON.stringify(taskLocJson, null, 2));
+
+        // Check that task.loc and task.loc.json versions match
+        if ((taskJson.version.Major !== taskLocJson.version.Major) || 
+            (taskJson.version.Major !== taskLocJson.version.Major) || 
```

Comment:
You're checking the major version 3 times, should be minor and patch

Reply:
Fixed.